packages/flutter_tools/lib/src/base/config.dart

```diff
@@ -92,9 +103,24 @@ class Config {
   //
   // Note that this is different from FileSystemUtils.homeDirPath.
   static String _userHomePath(Platform platform) {
-    final String envKey = platform.operatingSystem == 'windows'
-      ? 'APPDATA'
-      : 'HOME';
+    final String envKey =
```

Comment:
while you are here (and if you're going to reformat it...), change this to `platform.isWindows` ? ... : ...